internal/models/k8s.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+import (
+	"k8s.io/client-go/kubernetes"
+)
+
+type KubeData struct {
+	Clientset  *kubernetes.Clientset
+	Namespace  string
+	TargetName string
+	TargetType string
+}
+
+type DataFromK8sAPI struct {
+	KubeData       *KubeData
+	DeploymentData *DeploymentData
+}
```

Comment:
These structures are used to obtain some data from the k8s. So, I suppose that it will be good to combine these structures and methods for them into one single package and move it from `modules` to `pkg/k8s` for example.

Reply:
resolved:created new package k8s and combined methods, interface,struct  https://github.com/ScienceSoft-Inc/integrity-sum/pull/48/commits/aac49b5bba9e1e960a652315e53b3721b614da4e